src/components/UserInfo/UserInfo.scss

```diff
@@ -1 +1,3 @@
-//
```

Comment:
There doesn't appear to be anything outright wrong with your code. However, it is advisable to include additional styling for a better user interface experience.